docs/getting-started-guides/scratch.md

```diff
@@ -336,7 +336,7 @@ brctl delbr docker0
 
 The way you configure docker will depend in whether you have chosen the routable-vip or overlay-network approaches for your network.
 Some docker options will want to think about:
-  - create your own bridge for the per-node CIDR ranges, and set `--bridge=cbr0` and `--bip=false`.  Or let docker do it with `--bip=true`.
+  - create your own bridge for the per-node CIDR ranges, and set `--bridge=cbr0`.  Or let docker do it with something like: `--bip=$NODE_X_POD_CIDR`.
```

Comment:
I think mentioning --bip is more complex that simply documenting our cbr0 flag to kubelet and making those instructions good.  For example, --bip assumes that docker0 does not exist, but if it does exist with a different IP, docker fails to start.